docs/main/user-guides/mayastor.md

```diff
@@ -6,6 +6,24 @@ keywords:
 description: Mayastor documentation is hosted and actively maintained at https://mayastor.gitbook.io/introduction/
 ---
 
-:::note
-   This page has moved. Mayastor documentation is hosted and actively maintained at https://mayastor.gitbook.io/introduction/
+### Install and Setup
+
+:::warning
+Mayastor is incompatible with NDM (openebs-ndm) and cStor (cstor). Installing or upgrading Mayastor with `--set mayastor.enabled=true` will either not deploy LocalPV Provisioner and NDM or will remove them (if they already exist).
+
+However, installing Mayastor will not affect any preexisting LocalPV volumes.
 :::
+
+Before deploying and using Mayastor ensure that all the [prerequisites](https://mayastor.gitbook.io/introduction/quickstart/prerequisites) are met.
+
+- To install OpenEBS in a new cluster, use the following steps.
```

Comment:
Missing Mayastor here. Maybe, "To install Mayastor from OpenEBS charts in a new cluster.."

Reply:
Updated the text.